vocabularies/Org.OData.JSON.V1.xml

```diff
@@ -68,10 +132,57 @@
+        <Annotation Term="Core.LongDescription">
+          <String>Extracts a JSON value, such as an array, object, or a JSON scalar value (string, number, boolean, or `null`) from the `input` JSON value:
+- If `path` only consists of the root identifier followed by name and index selectors, it returns the identified single node within `input`, or `null` if no node is identified. 
+- If `path` potentially identifies multiple nodes within `input` (by using descendant, wildcard, union, array subset, or filter selectors), it returns an array containing the identified nodes, or an empty array if no node is identified. 
+- If `input` is not a valid JSON value, the function returns `null`.
+- If `path` is `null`, not a valid [JSONPath expression](#Path), or does not match the structure of `input` (for example applying an index selector to a scalar value), the function returns `null`. 
+          </String>
+        </Annotation>
+        <Parameter Name="input" Type="JSON.JSON">
+          <Annotation Term="Core.Description" String="JSON input" />
+        </Parameter>
+        <Parameter Name="path" Type="JSON.Path">
+          <Annotation Term="Core.Description" String="JSONPath expression to be applied to value of `expr`" />
+        </Parameter>
+        <ReturnType Type="JSON.JSON">
+          <Annotation Term="Core.Description" String="JSON value resulting from applying `path` to `input`" />
+        </ReturnType>
+      </Function>
+
+      <Function Name="value" IsBound="true">
+        <Annotation Term="Core.Description" String="Query a stream value of media type `application/json`, returning an OData primitive value" />
+        <Annotation Term="Core.LongDescription">
+          <String>Extracts a single OData primitive value from the `input` JSON value:
+- If `path` only consists of the root identifier followed by name and index selectors and identifies a single scalar JSON value (string, number, boolean, or `null`) within `input`, it returns the identified single value cast to an OData primitive value (see below).
+- If `path` identifies multiple nodes within `input` (by using descendant, wildcard, union, array subset, or filter selectors), identifies an object or array, or does not identify any node, the function returns `null`.
+- If `input` is not a valid JSON value, the function returns `null`.
+- If `path` is `null`, not a valid [JSONPath expression](#Path), or does not match the structure of `input` (for example applying an index selector to a scalar value), the function returns `null`.
+
+If a single non-null scalar JSON value is identified by `path` within `input`, the function returns that value as an OData primitive value of type
+- `Edm.String` if the value is a JSON string
+- `Edm.Boolean` if the value is `true` or `false`
+- `Edm.Decimal` with unspecified precision and floating scale if the value is a JSON number
```

Comment:
[RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259#section-6) and [RFC 7493](https://datatracker.ietf.org/doc/html/rfc7493#section-2.2) both recommend only sending number values that fit into IEEE 754 binary64 (double precision) numbers, we could use `Edm.Double` here.